server/app/views/applicant/NavigationFragment.html

```diff
@@ -355,3 +355,20 @@
   rel="noopener noreferrer"
   >civiform-dev@exygy.com</a
 >
+
+<section
+  th:fragment="adminPreviewBackButton(goBackUrl)"
+  class="grid-container section"
+  th:if="${userIsAdmin}"
+>
+  <div class="grid-row">
+    <div class="tablet:grid-col-8 usa-prose">
+      <a th:href="${goBackUrl}">
```

Comment:
nitpick (non-blocking): what do you think of naming this var `goBackToAdminUrl` to match the other ones and to make it more clear in here that it's returning you to an admin view?